features/_your_details/your_details_disabling_minimum_contributions.feature

```diff
@@ -6,7 +6,7 @@ Feature: Employment contribution
 
   @no-javascript
   Scenario Outline: Annual salary rate below £5,876 with minimum employer contributions
-    Given that I am on the WPCC homepage
+    Given I am on step 1 of the WPCC homepage
```

Comment:
leave all the cucumber stuff to me! 😅  I'm hoping I get on top of it before things get even more unwieldy!